recipes/libxml2/all/conanfile.py

```diff
@@ -255,9 +257,13 @@ def package_info(self):
             self.env_info.PATH.append(bindir)
 
         if self.settings.os == "Linux" or self.settings.os == "Macos":
-            self.cpp_info.system_libs.append('m')
+            self.cpp_info.system_libs.append("m")
         if self.settings.os == "Windows":
-            self.cpp_info.system_libs.append('ws2_32')
-        self.cpp_info.names["cmake_find_package"] = "LibXml2"
-        self.cpp_info.names["cmake_find_package_multi"] = "LibXml2"
+            self.cpp_info.system_libs.append("ws2_32")
+        if self.options.threads:
+            if self.settings.os == "Linux":
+                self.cpp_info.system_libs.append("pthread")
+
         self.cpp_info.names["pkg_config"] = "libxml-2.0"
+        self.cpp_info.filenames["cmake_find_package"] = "libxml2"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "libxml2"
```

Comment:
```suggestion
        self.cpp_info.names["cmake_find_package"] = "LibXml2"
        self.cpp_info.names["cmake_find_package_multi"] = "libxml2"
```

I would suggest this instead.

module file: https://cmake.org/cmake/help/v3.18/module/FindLibXml2.html
config file: `libxml2-config.cmake`
It's true that there is no imported target in config file. So let keep default name for target of config file (or same name than before?).

Reply:
Yup, I'll make the change and add a fixme about existence of libxml2 + missing `LibXml2::xmllint` executable target.